package.json

```diff
@@ -22,7 +22,8 @@
     "react-query": "3.9.4",
     "react-router": "5.2.1",
     "react-router-dom": "5.3.0",
-    "recoil": "0.6.1"
+    "recoil": "0.6.1",
+    "yarn": "^1.22.17"
```

Comment:
Don't use fixed versions (with ^)

Reply:
We needn't the `yarn` as a dependency of project. You need to remove this line.